drivers/src/persistent.rs

```diff
@@ -164,6 +231,12 @@ pub struct PersistentData {
 
     pub idevid_csr: IdevIdCsr,
     reserved10: [u8; memory_layout::IDEVID_CSR_SIZE as usize - size_of::<IdevIdCsr>()],
+
+    #[cfg(feature = "fmc")]
+    pub fmc_alias_csr: FmcAliasCsr,
+
+    #[cfg(feature = "fmc")]
+    reserved11: [u8; memory_layout::FMC_ALIAS_CSR_SIZE as usize - size_of::<FmcAliasCsr>()],
```

Comment:
When the `fmc` is not enabled, a slice of `memory_layout::FMC_ALIAS_CSR_SIZE` should still be reserved so `PersistentData` is the same size / shape between image types.